client/src/modules/plan/planner/idea/idea.html

```diff
@@ -0,0 +1,25 @@
+<div class="idea-container flex-col">
+  <div class="idea-image">
+    <button class="btn btn-secondary" ng-click="$ctrl.EventService.promoteEvent($ctrl.ideaId)">Schedule</button>
```

Comment:
in the future we may need to pull these into the controller in case we ever want to do error handling